VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 |---------|-------------|-----------------|----------------------|
+| 4.0.0-beta.1  | 4.6.1      | 5.1.0           | 3.2.1               |
```

Comment:
This should be 3.2.2 and and iOS 4.7.0

Reply:
ohh right, I forgot I updated that afterwards too

Reply:
great catch!

Reply:
updated!